build.gradle.kts

```diff
@@ -24,9 +24,10 @@ allprojects {
     }
 }
 
-configure<JavaPluginExtension> {
-    sourceCompatibility = JavaVersion.VERSION_11
-    targetCompatibility = JavaVersion.VERSION_11
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(17)
```

Comment:
So this is the new way to go? 😄

Reply:
Yes but it requires Gradle 8. However, our plugins are broken...